Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.cpp

```diff
@@ -59,11 +83,21 @@ namespace AzToolsFramework
                 settings.m_metadata.Add(static_cast<AZ::JsonEntityIdSerializer::JsonEntityIdMapper*>(&entityIdMapper));
                 settings.m_metadata.Add(&entityIdMapper);
 
-                if ((flags & StoreInstanceFlags::StripDefaultValues) != StoreInstanceFlags::StripDefaultValues)
+                if ((flags & StoreFlags::StripDefaultValues) != StoreFlags::StripDefaultValues)
                 {
                     settings.m_keepDefaults = true;
                 }
 
+                AZStd::string scratchBuffer;
```

Comment:
PR Bonus Points!

Reply:
I found the default issue reporter did this. Seemed like a good idea to not allocate a string on each hit.